### DISCUSSION TITLE
Should Discussions supersede Question issues?

### DISCUSSION THREAD

**Discussion by StephanTLavavej** [microsoft/STL#1117] [General] [answered]
Our issue template for questions, [`.github/ISSUE_TEMPLATE/question.md`](https://github.com/microsoft/STL/blob/4e97255b3595a01f5268b98fad397e8579d14ed2/.github/ISSUE_TEMPLATE/question.md), was created before Discussions. Should we remove this issue template, or possibly convert it to a wiki page about what's on-topic?

**Comment by cbezault** [comment, marked as answer]
Questions should definitely get subsumed into discussions. I think we already cover what is on-topic sufficiently to steer the discussions page, at least for now.

**Reply by barcharcraz** [reply]
wow is this a new feature?

**Reply by CaseyCarter** [reply]
It's in beta, IIUC.
